layers/meta-resin-raspberrypi/recipes-bsp/u-blox-modeswitch/u-blox-modeswitch.bb

```diff
@@ -0,0 +1,33 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+SUMMARY = "Switch u-blox modem from RNDIS to ECM mode"
+
+LICENSE="MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+DESCRIPTION = "Package contains udev rule and systemd service for switching u-blox modems from RNDIS to ECM mode. \
```

Comment:
This doesn't seem to be Raspberry Pi specific in any way. Why are we adding it in `resin-raspberrypi`?

Reply:
This problem shows up on a client's board that is based on RPi CM3. 
We didn't yet figured out if the problem is from the kernel(RPi specific) or other component like openvpn, which is not RPi specific.
I think that this fix should only be available for the RPi boards at this moment.
That's why I chose to do the fix in resin-raspberrypi, until we figure out what is the root cause.